commons/utils/src/test/java/org/operaton/commons/utils/EnsureUtilTest.java

```diff
@@ -41,14 +41,8 @@ void ensureNotNull() {
   @Test
   void shouldFailEnsureNotNull() {
     String string = null;
-
-    try {
-      EnsureUtil.ensureNotNull("string", string);
-      fail("Expected: IllegalArgumentException");
-
-    } catch(IllegalArgumentException e) {
-      // happy path
-    }
+    assertThatThrownBy(() -> EnsureUtil.ensureNotNull("string", string), "Expected: IllegalArgumentException")
```

Comment:
Also here: The description does not add value, thus remove. Therefore in general: Remove the obvious messages that have been added to `fail()`. It is one of the benefits of this refactoring that this is not needed.